Sources/Property.swift

```diff
@@ -678,3 +711,71 @@ public final class MutableProperty<Value>: MutablePropertyProtocol {
 		observer.sendCompleted()
 	}
 }
+
+extension MutableProperty {
+
+	/// Returns a `BindingTarget` that invokes the given setter to generate a
+	/// new value for this property.
+	///
+	/// Consider the following example:
+	///
+	/// ```
+	/// let personProperty = MutableProperty(Person(firstName: "Steve", lastName: "McQueen"))
+	/// let firstNameBinding = personProperty.bindingTarget {
+	///         return Person(firstName: $1, lastName: $0.lastName)
```

Comment:
Can you name these arguments? I think for documentation purposes that'll be a lot clearer.